frontend/src/components/Utils/time.test.js

```diff
@@ -27,6 +27,24 @@ test('other date timestamp format', () => {
  expect(actualSingapore).toEqual(expectedSingapore);
 });
 
+test('new york date timestamp format, no weekday', () => {
+  // Month parameter is zero indexed so it's actually the 10th month.
+  const testDate = new Date(Date.UTC(2020, 9, 3, 14, 19, 4, 23)).getTime();
+  const expected = 'October 3, 2020';
+  const actual = utils.timestampToLongDate(testDate);
+  expect(actual).toEqual(expected);
+ });
+ 
+ test('other date timestamp format, no weekday', () => {
+  const testDate = new Date(Date.UTC(2020, 7, 23, 2, 3, 2, 4)).getTime();
+  const expectedCentral = 'August 22, 2020';
+  const expectedSingapore = 'August 23, 2020';
+  const actualCentral = utils.timestampToLongDate(testDate, TZ_CHICAGO);
+  const actualSingapore = utils.timestampToLongDate(testDate, TZ_SINGAPORE);
+  expect(actualCentral).toEqual(expectedCentral);
+  expect(actualSingapore).toEqual(expectedSingapore);
+ });
+
```

Comment:
This could be totally wrong so getting @keiffer01, @Hiram-Silvey, or @beiebi0  's opinion would be a very good idea. 

Based on feedback I got in the past, it seems like most of these tests (ones you wrote in this PR and similar others in test.js) are testing a single call to a function in the moment library. So because there is no additional logic (your own conditionals or a series of library functions called in succession), tests may not be needed for these cases.